r/src/py-to-r.cpp

```diff
@@ -21,6 +21,38 @@
 
 #include <arrow/c/bridge.h>
 
+// [[arrow::export]]
+double external_pointer_addr_double(SEXP external_pointer) {
+  // potentially lossy conversion to double needed for the current
+  // implementation of import/export to Python
+  return reinterpret_cast<uintptr_t>(R_ExternalPtrAddr(external_pointer));
```

Comment:
Can this be something like ?

```c
double out;
void* ptr = R_ExternalPtrAddr(external_pointer);
memcpy(&out, R_ExternalPtrAddr(external_pointer), sizeof(double));
return out;
```

Reply:
It seems to be what is done in `external_pointer_addr_integer64()` below. Maybe if this returned a `cpp11::doubles` ?

Reply:
I see. It sort of feels wrong though.